src/webgpu/web_platform/external_texture/video.spec.ts

```diff
@@ -304,26 +303,13 @@ TODO: Make this test work without requestVideoFrameCallback support (in waitForN
 
       const commandBuffer = useExternalTexture();
       t.expectGPUError('validation', () => t.device.queue.submit([commandBuffer]), false);
-      t.expect(!externalTexture.expired);
 
       if (sourceType === 'VideoFrame') {
         (source as VideoFrame).close();
         const commandBuffer = useExternalTexture();
         t.expectGPUError('validation', () => t.device.queue.submit([commandBuffer]), true);
-        t.expect(externalTexture.expired);
       }
     });
-
-    if (sourceType === 'VideoElement') {
-      // Update new video frame.
-      await waitForNextFrame(videoElement, () => {
-        // VideoFrame is updated. GPUExternalTexture imported from HTMLVideoElement should be expired.
-        // Using the GPUExternalTexture should result in an error.
-        const commandBuffer = useExternalTexture();
-        t.expectGPUError('validation', () => t.device.queue.submit([commandBuffer]), true);
```

Comment:
Why does this need to be removed? Seems still valid though I know more work is needed on these tests due to the recentish spec changes.

Reply:
You're right. I planned to add video element expire cases in following PR but this piece is correct and no need to remove.
Let me add it back.